src/components/sales/SearchBar.tsx

```
@@ -105,14 +114,16 @@ export default function SearchBar({ value, onChange }: Props) {
 
       {scannerVisible && (
         <div className="fixed inset-0 bg-black bg-opacity-75 z-50 flex items-center justify-center">
-          <div className="bg-white rounded-lg p-6 w-full max-w-md relative">
+          <div className="bg-white dark:bg-gray-900 rounded-lg p-6 w-full max-w-md relative">
             <button
               onClick={stopScanner}
               className="absolute top-3 right-3 text-gray-500 hover:text-gray-700"
             >
               <X className="w-5 h-5" />
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)